# SmoothConfig
smoother_type: "Spline"  # "Spline" or "Discrete"
resmooth_threshold: 10.0
behind_distance: 5.0

# ExpectDistanceConfig
dis_base: 4.0
dis_speed_factor: 0.5
dis_max: 8.5
dis_min: 4.0
dis_offset: 3.55

#V2VConfig
v2v_acc_filter_cutoff_freq:  2.0
v2v_delay_threshold: 0.1
v2v_use_acc_threshold:  0.15

# TrackManagerConfig
target_use_global_pose: true
track_interval: 0.1
max_track_size: 3000
timeout_threshold: 1.0
distance_filter_cutoff_freq: 2.0
enable_mpc_adapter: true
